.vortex/tooling/src/helpers.php

```
@@ -660,14 +742,15 @@ function acli_resolve(): string {
 
-  task(sprintf('Downloading the Acquia CLI "%s" to "%s".', $version, $bin));
-  $response = request($url, ['method' => 'GET', 'save_to' => $bin, 'timeout' => 120]);
-  if (!$response['ok']) {
-    @unlink($bin);
-    fail('Failed to download the Acquia CLI from "%s": %s', $url, $response['error'] ?? 'Unknown error');
-  }
+  TASK(sprintf('Downloading the Acquia CLI "%s" to "%s".', $version, $bin), sprintf('Downloaded the Acquia CLI "%s" to "%s".', $version, $bin), function () use ($url, $bin): void {
+    $response = request($url, ['method' => 'GET', 'save_to' => $bin, 'timeout' => 120]);
+    if (!$response['ok']) {
+      @unlink($bin);
+      throw new \RuntimeException(sprintf('Failed to download the Acquia CLI from "%s": %s', $url, $response['error'] ?? 'Unknown error'));
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '749', column '17'). (undefined)

(MissingImport)